Common/itkComputeDisplacementDistribution.hxx

```diff
@@ -53,7 +53,7 @@ ComputeDisplacementDistribution<TFixedImage, TTransform>::ComputeDisplacementDis
 
   /** Threading related variables. */
   this->m_UseMultiThread = true;
-  this->m_Threader = ThreaderType::New();
+  this->m_Threader = MultiThreaderBase::New();
```

Comment:
Might use thread pools, so might improve the performance of ComputeDisplacementDistribution 👍